infrastructure/deploy/ci_ingress.tf

```diff
@@ -0,0 +1,18 @@
+resource "aws_security_group_rule" "data_refinery_ci_nomad" {
+
+  type = "ingress"
+  from_port = 4646
+  to_port = 4646
+  protocol = "tcp"
+  cidr_blocks = ["${var.host_ip}/32"]
```

Comment:
This the whole /32 - I think this can be a /1. Is there a /1?

Reply:
32 means the entire IP address is relevant.

Reply:
Derp.